packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -26,6 +26,46 @@ export const toggleGroupControl = ( {
 
 	${ toggleGroupControlSize( size ) }
 	${ ! isDeselectable && enclosingBorders( isBlock ) }
+
+	@media not ( prefers-reduced-motion ) {
+		&.is-animation-enabled::before {
+			transition-property: transform, border-radius;
+			transition-duration: 0.2s;
+			transition-timing-function: ease-out;
+		}
+	}
+
+	&::before {
```

Comment:
Do you think that adding `contain: strict` to the indicator could potentially squeeze some extra rendering performance? It probably won't hurt anyway?

Reply:
Based on what I've read about the property, I'm not sure that'd do anything for us in the indicator. I could imagine _maybe possibly_ doing something in the control container itself, though I doubt there's any tangible benefit and I'm worried that it could cause some headaches or obscure bugs down the road.

I'd prefer not to use it for now, at least until I or someone else becomes knowledgeable enough on this method to determine its effects with confidence, and only if there's a clear benefit.